src/Flicking.ts

```diff
@@ -651,7 +652,9 @@ class Flicking extends Component<{
         const cloneIndex = Math.floor(val / prevPanelCount);
```

Comment:
`visibleIndexes` is like the latest index. Is it right to use `prevPanelCount`?

If `circular` is used, the `cloneIndex` will be very large.

Reply:
`val` is using an absolute index based on previous panel count, see line 645

Reply:
In the test, beforeSync was called first, so I thought that the indexes of visiblePanels were all changed.

is correct that cloneIndex and index do not change even after beforeSync?

Reply:
Well, that should be handled differently. I'll send a PR tomorrow.